README.md

```diff
@@ -1,4 +1,3 @@
-<!DOCTYPE html>
```

Comment:
this still shows on the github.io page where it should not.

Reply:
The live website is pulling from the master branch. Once this change gets merged, the website will no longer show this.